tests/template/elegant_test.py

```diff
@@ -27,7 +27,7 @@ def test_file_iterator():
     from sirepo.template.lattice import LatticeUtil
     from pykern.pkunit import pkeq
     data = _find_example('bunchComp - fourDipoleCSR')
-    v = LatticeUtil(data, _elegant()._SCHEMA).iterate_models(
+    v = LatticeUtil(data, _elegant().SCHEMA).iterate_models(
```

Comment:
I don't think we should be exporting this constant for this purpose. A unit test can reach inside for _SCHEMA, but really it should ask sim_data.

Reply:
It is really exported for use [here](https://github.com/radiasoft/sirepo/pull/3595/files#diff-687323f8346dfdf343f1eee4795f4492f47f05a6fae712ee4b920bd63897e68aR579). That ok?

Reply:
Do you mean [here](https://github.com/radiasoft/sirepo/pull/3595/files#diff-24496517cc6cc5369fc69a13a07158c876115ef9eec8ceb098a25b374b083f7fL748)? If so, this is not exported, it's part of an object. There's not particular reason to use the constant vs calling sim_data or simulation_db in a test. It's not a performance issue.

Reply:
Sorry - not sure why my link didn't have a better line number. It is exported for use inside of [`template.common.stateful_compute_dispatch`](https://github.com/radiasoft/sirepo/pull/3595/files#diff-687323f8346dfdf343f1eee4795f4492f47f05a6fae712ee4b920bd63897e68aR592)